src/Uno.CrossTargetting.props

```diff
@@ -137,6 +137,10 @@
 	<Target Name="CommonOverrideNuget" AfterTargets="AfterBuild" DependsOnTargets="BuiltProjectOutputGroup" Condition="'$(UnoNugetOverrideVersion)'!='' and '$(UseCommonOverridePackage)'!=''">
 		<PropertyGroup>
 			<_OverrideTargetFramework>$(TargetFramework)</_OverrideTargetFramework>
+
+			<!-- Current nuget publishing uses netX.0-android30.0 -->
+			<_OverrideTargetFramework Condition="$(_OverrideTargetFramework.EndsWith('.0-android'))">$(_OverrideTargetFramework)30.0</_OverrideTargetFramework>
```

Comment:
Can we instead fix the original occurrences?

Reply:
I'm not sure if we can just yet. The problem here is that we'll be trying to build with an SDK version that is latest, but for which we want to advertise we're building against. Let's see what we can do when we bump to Level 33.